SECURITY.md

```diff
@@ -0,0 +1,69 @@
+# Security Policy
+
+## Supported Versions
+
+We recommend to only use the most recent minor release.
+ 
+## Reporting a Vulnerability
+
+**Please do not file a public ticket** mentioning the vulnerability.
+
+To disclose a vulnerability, please submit a `Security Advisory` via the `Security` tab on the impacted repository.
+
+If a repository doesn't have the proper security reporting set up, please email `security@celestia.org` to report the vulnerability.
+ 
+## Disclosure Policy
+Vulnerability details may be shared with third parties after the vulnerability
+has been fixed and the program owner has provided permission to disclose or
+after 120 days from submission, whichever is sooner.
+
+### Scope
+There is nothing considered out-of-scope for testers and researchers following
+the rules outlined in this policy.
+
+### Rewards
+Celestia Labs has no formal reward policy and researchers should not expect a reward
+for discovering a vulnerability. Celestia Labs is nonetheless grateful for all
+legitimate discoveries of vulnerabilities, and is happy to acknowledge the
+vulnerability and the researchers after a fix has been widely deployed.
+
+
+### Official Communication Channel
+???
```

Comment:
Is this supposed to be the official comms channel between the reporting party and Celestia? That should always be security@celestia.org for the sake of simplicity IMO (unless agree with the party differently).
Celestia's official comms channels can be found on the homepage under community: https://celestia.org/

Reply:
This should be where we plan to communicate security updates publicly. 

So one piece will be that all communication about security reports would come from the `security@celestia.org` email, but then where do we want to say we will announce security updates? Twitter? Discord? Github?

Reply:
made a general update

Reply:
Notes:
1. just fyi: security@ is a Google group to receive emails, so it can't be the _sender_of emails
2. I think saying "celestia.org" is the official channel for announcements isn't good enough as that's too vague. I think there should probably be a concrete place for people to receive security advisories - would this be via the GitHub security advisories feature by default?

Reply:
> would this be via the GitHub security advisories feature by default

I don't think so, those are for reporting to us, not us reporting out.  Only owners, admins, and the security team have access to see those.

> just fyi: security@ is a Google group to receive emails, so it can't be the sender of emails afaik

You can actually set it as an alias, so you can send emails from a google group https://support.google.com/googlecloud/answer/10635789?hl=en

>  I think there should probably be a concrete place for people to receive security advisories

I agree, should we just pick one? Say discord?

Reply:
>I don't think so, those are for reporting to us, not us reporting out. Only owners, admins, and the security team have access to see those.

>I agree, should we just pick one? Say discord?

https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/publishing-a-repository-security-advisory
?

>You can actually set it as an alias, so you can send emails from a google group https://support.google.com/googlecloud/answer/10635789?hl=en

I'm unsure if it's good practice to be sending emails from security@ though; rather they should be attributed to an individual on the team. But I don't have a strong opinion.

Reply:
> https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/publishing-a-repository-security-advisory

TIL. then yea, we could use this.

Reply:
I can update once Jeremey reviews and include any additional updates

Reply:
Regarding the official communications channel - does GitHub also not have a built-in way to respond to security reports inside of GitHub, and if so should that be mentioned?